Jenkinsfile

```diff
@@ -1,3 +1,6 @@
-@Library('defra-library@5') _
+@Library('defra-library@v-6')
```

Comment:
This line will need to end with an underscore i.e. `@Library('defra-library@v-6') _` to load the global vars.

Reply:
_ added